packages/ui-components/src/components/layout/Table.stories.ts

```diff
@@ -0,0 +1,222 @@
+        avatar: 'https://i.pravatar.cc/150?img=2',
+        verified: false,
+        company: 'Microsoft',
+        role: 'User'
+      },
+      {
+        id: 3,
+        name: 'Michael O’Brien',
+        email: 'michael.obrien@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=3',
+        verified: true,
+        company: 'Amazon',
+        role: 'Manager'
+      },
+      {
+        id: 4,
+        name: 'Sarah Kim',
+        email: 'sarah.kim@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=4',
+        verified: false,
+        company: 'Netflix',
+        role: 'Developer'
+      },
+      {
+        id: 5,
+        name: 'Carlos Rodriguez',
+        email: 'carlos.rodriguez@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=5',
+        verified: true,
+        company: 'Adobe',
+        role: 'Designer'
+      },
+      {
+        id: 6,
+        name: 'Emily Johnson',
+        email: 'emily.johnson@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=6',
+        verified: false,
+        company: 'Salesforce',
+        role: 'Admin'
+      },
+      {
+        id: 7,
+        name: 'Fiona Chen',
+        email: 'fiona.chen@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=7',
+        verified: true,
+        company: 'Spotify',
+        role: 'User'
+      },
+      {
+        id: 8,
+        name: 'David Smith',
+        email: 'david.smith@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=8',
+        verified: false,
+        company: 'Tesla',
+        role: 'Manager'
+      },
+      {
+        id: 9,
+        name: 'Aaliyah Jackson',
+        email: 'aaliyah.jackson@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=9',
+        verified: true,
+        company: 'Facebook',
+        role: 'Developer'
+      },
+      {
+        id: 10,
+        name: 'Mohamed Ali',
+        email: 'mohamed.ali@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=10',
+        verified: false,
+        company: 'Twitter',
+        role: 'Designer'
+      },
+      {
+        id: 11,
+        name: 'Sophia Lee',
+        email: 'sophia.lee@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=11',
+        verified: true,
+        company: 'Snap Inc.',
+        role: 'Admin'
+      },
+      {
+        id: 12,
+        name: 'Liam Brown',
+        email: 'liam.brown@example.com',
+        avatar: 'https://i.pravatar.cc/150?img=12',
+        verified: false,
+        company: 'Apple',
+        role: 'User'
+      }
+    ],
+    overflowCells: false,
+    onRowClick: (item: TableItemType) => console.log('Row clicked', item),
+    roles: ['Admin', 'User', 'Guest']
+  }
+}
+
+export const WithButtons: StoryObj = {
+  ...Default,
+  args: {
+    ...Default.args,
+    buttons: [
+      {
+        icon: DocumentDuplicateIcon,
+        label: 'Duplicate',
+        action: (item: TableItemType) => console.log('Duplicate', item),
```

Comment:
Instead of console logging, use `action()` from `'@storybook/addon-actions'` so that these messages can be actually seen within storybook (google for more info)